components/dashboard/src/settings/EnvironmentVariables.tsx

```diff
@@ -145,26 +176,32 @@ export default function EnvVars() {
                 }
             }
         }
+        if (!variable.id && envVars.some(v => v.name === name && v.repositoryPattern === pattern)) {
```

Comment:
**question**: This may be rhetorical but is it ok to add overlapping scopes like the following?

<img width="588" alt="Screenshot 2021-04-26 at 4 02 10 PM" src="https://user-images.githubusercontent.com/120486/116087002-18c24200-a6a9-11eb-913b-a6616d003bc8.png">

Reply:
Good catch! Yes, this is expected. 🙂